portal.py

```diff
@@ -23,12 +23,13 @@
     with open(gpu_dir, "w") as gpu_flag:
         gpu_flag.write("-1")
 
-
 if os.getenv("IS_ELECTRON"):
     os.system("npm run portal-build")
 else:
-    if (sys.platform == "win32"):
+    if sys.platform == "win32":
         os.system("python portal-build/run.py")
+    elif sys.platform == "linux":
+        os.system("python3 portal-build/run.py")
     else:
         # Non windows
         os.system("python3 -m portal-build/run")
```

Comment:
Checking the documentation in https://docs.python.org/3/library/sys.html , I noticed there wasn't a mac case. Does your else branch (last one) account for mac?

Reply:
yeap the last one accounts for mac